docs/abi.md

```diff
@@ -53,7 +53,7 @@ Two specific sets of integer values are used in multiple places in the ABI:
 Each Oak Module must expose the following **exported function** as a
 [WebAssembly export](https://webassembly.github.io/spec/core/syntax/modules.html#exports):
 
-- `oak_main: () -> u32`: Invoked when the Oak Manager executes the Oak Node.
+- `oak_main: (u64) -> u32`: Invoked when the Oak Manager executes the Oak Node.
   This function should perform its own event loop, reading incoming messages
```

Comment:
Mention what the parameter is in this paragraph.

Reply:
Done.